syntax = "proto3";

// 网关机的rpc server定义
// cd gateway/rpc 下执行protoc去编译
service State {
    rpc CancelConn (StateRequest) returns (StateResponse);
    rpc SendMsg (StateRequest) returns (StateResponse);
}

message StateRequest{
    string endpoint = 1;
    uint64 connID = 2;
    bytes  data = 3;
}
  
message StateResponse {
    int32 code = 1;
    string msg = 2;
}